type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in dt_to_decimal: Name 't_sec' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_create: No attribute 'assertIsInstance' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_delete: No attribute 'cs' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in handle_events: Name 'emsg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_receive_timeout: No attribute 'mock_channel' on Type[] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_request: Can't assign attribute 'return_value' on str [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in excluded_type: Name 't' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _extract_when: Name 'when' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_jsonreports_search_with_invalid_period_start: Function ast.literal_eval was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 8, in test_jsonreports_search_with_invalid_id: Name 'actual_result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in mock_datetime_utcnow: No attribute 'AndReturn' on datetime.datetime [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in get_instance_type: Name 'inst_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _verify_basic_validity: Name 'field_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in service: Name 'parts' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in host: Name 'parts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in save_exists: Name 'images' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in save_delete: Name 'values' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in host: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 3, in service: Name 'parts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _create_child_logger: Name 'queue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _create_timed_rotating_logger: No attribute 'handlers' on logging.Formatter [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in listdir: Name 'objects' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in read: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in th: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in store: Name 'report' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _get_new_launches: Name 'filters' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in cell_and_compute: Name 'usage_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in store_results: Name 'values' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in make_json_report: Name 'report' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in latest_raw_for_request_id: No attribute 'request_id' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in host: Name 'raw' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in handle_events: Name 'key' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in _process_block: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in engine: Name 'engine' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in sessionmaker: Name 'maker' is not defined [name-error]
File ""dummpy_input_file"", line 9, in sessionmaker: Invalid keyword argument bind to function unsolveable.sessionmaker [wrong-keyword-args]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_fields_match: Name 'match_code' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_get_instance: No attribute '_fake_instance' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in test_do_events_of_a_single_service: No attribute 'assertEqual' on List[Dict[str, str]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_do_hosts: No attribute 'mox' on List[Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_glance_rawdata: No attribute 'save' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_image_usage: No attribute 'save' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: Name 'text' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in revert: No attribute '_revert' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in execute: No attribute '_functor' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in execute: Name 'l' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in test_task_detail_save: No attribute 'update_flow_details' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in setUp: Name 'conn' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_rebind_unknown_kwargs: No attribute 'assertEqual' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in run_one: No attribute 'values' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in revert_one: No attribute 'append' on Callable [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in run_fail: No attribute 'values' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_fetch_optional_args_found: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_get_filter_args_bad_uuid: Name 'fake_model' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_get_filter_args_bad_min_value: Name 'fake_request' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in test_list_usage_exists_with_received_min: No attribute 'status_code' on Dict[str, Dict[str, Any]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_search_by_field_for_nova: Name 'fake_request' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_not_empty: Name 'root' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_find: Name 'root' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_reply_notify_invalid: No attribute 'assertRaises' on Dict[str, Union[Dict[nothing, nothing], str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in on_failure: No attribute '_attempts' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _get_next_value: Name 'remaining' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in on_failure: No attribute '_on_failure' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in fetch_action: Name 'metadata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in transition: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in connection_details: No attribute '_conn' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _format_time: Function date.strftime was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_copy_listeners: No attribute 'notifier' on Callable[[Any, Any], Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in compile_bad: Name 'engine' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _make_queue: Name 'queue_name' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in make_save_book: No attribute 'save_logbook' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in execute: No attribute 'provides' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in request_to_dict: No attribute 'task' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in server: No attribute 'Server' on Callable[[Any, Any], Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_creation: Name 'master_mock_calls' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_process_request: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_reset_unknown_task: Name 's' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_task_metadata_update_with_none: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_save_multiple_results: Name 's' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in test_logbook_delete: No attribute 'destroy_logbook' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in wrapper: No attribute '_storage_ensured' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_do_request: Name 'raw' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in test_send_status: No attribute 'send_status' on Dict[str, int] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in test_send_status_batch_accepts_post_when_version_is_0: No attribute 'mox' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 21, in test_send_status_batch_not_found: No attribute 'mox' on Dict[str, Dict[Any, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in test_list_usage_launches_for_glance: No attribute 'assertEqual' on Dict[str, int] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_cause: Name 'e' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_raise_with: Name 'e' is not defined [name-error]
File ""dummpy_input_file"", line 16, in test_raise_with: No attribute 'cause' on Exception [attribute-error]
File ""dummpy_input_file"", line 16, in test_raise_with: No attribute 'cause' on None [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_no_looping: No attribute 'append' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_flow_detail_save: No attribute '_get_connection' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in request: No attribute 'task_uuid' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _create_properties: No attribute 'type_definition' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in assert_get_attribute_host_functionality: Name 'configure_op' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_get_num_from_scalar_unit_size_negative: No attribute 'assertTrue' on Exception [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_get_num_from_scalar_unit_frequency_negative: No attribute 'assertTrue' on Exception [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_invalid_scalar_unit: No attribute 'assertRaises' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_template_not_exist: Name 'error' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_template_valid: No attribute 'fail' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in reviews: Name 'cmd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in daemon_thread: Can't assign attribute 'daemon' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _tpl_relationship_templates: Name 'topology_template' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 18, in _get_all_custom_defs: No attribute '_get_all_custom_defs' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __getitem__: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_start: No attribute 'master_mock' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in test_multi_different_topic_workers: No attribute 'get_worker_for_task' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _format_capture: Name 'name' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in execute: No attribute 'update_progress' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in revert: No attribute 'update_progress' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in execute: No attribute 'update_progress' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in execute: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in execute: No attribute 'update_progress' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _safe_header: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in fail_status: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 9, in display_feedback: Name 'report_value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in delete_file: Name 'ce' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_class: Name 'client_path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _failure_percentage: Name 'total' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 16, in collect_metrics: unsupported operand type(s) for item retrieval: 'start: float' and ''query': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Function BaseException.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
True,"File ""dummpy_input_file"", line 24, in run_command_status: Name 'nothing' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_version: No attribute 'version' on pkg_resources.Requirement [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in git_directories: Name 'out' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_sections: Name 'version' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in get_defs_file: Name 'versiondata' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in search: No attribute '_queries' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in setup_logging: Function logging.config.fileConfig was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _get_capability_properties_def_objects: No attribute 'get_properties_def_objects' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_get_image_name: No attribute 'cloud' on Image [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_list_hypervisors: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _cache_decorator: No attribute '__get__' on Any [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Can't assign attribute 'orig_message' on list [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __str__: No attribute 'log_inner_exceptions' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_StartRun_should_start_a_timer_and_save_the_execution_to_DB: Name 'run_manager' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_EndRun_should_get_timer_data_and_modify_a_record_in_DB: Name 'run_manager' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_GetURLsToVisit_filters_image_urls: No attribute 'url_manager' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_IsURL: No attribute 'url_manager' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_AddURLToDB_should_call_the_DB_instance_to_perform_the_work: No attribute 'core_mock' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_UserAbort_with_plugin_should_raise_and_exception: Name 'error_handler' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_Add_should_print_and_save_to_the_DB_a_non_OWTF_error: Name 'error_handler' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_activate_node_timeout: No attribute 'node' on Type[available_node_state] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_set_node_instance_info: No attribute 'node' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in _action_WAITFORRESP: No attribute 'dataLock' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in uri2options: No attribute 'startswith' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in InitPaths: Name 'MandatoryPaths' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in owtf_last_commit: Name 'commit_hash' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in next: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in associate_alternate_id: No attribute '_build_and_send_request' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_alternate_ids: Name 'tree' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_range_search_2: No attribute 'assertEqual' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_update_port: Name 'port' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_that_it_is_possible_to_define_headers_to_be_returned_by_the_server: No attribute 'set_response' on Dict[str, str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_setting_different_methods_to_the_same_path: Name 'get_response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_that_it_is_possible_to_specify_the_response_code: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in find_files_matching: Function bytes.split was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in arguments_are_valid: Name 'validArguments' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in PUT: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in MultipleReplace: Name 'Replace' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in scrub_output: Function Pattern.sub was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in InitExecutionRegistry: Name 'Target' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_inspect_machine_failed: No attribute 'node' on Type[inspect_failed_machine] [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_inspect_machine_wait: No attribute 'node' on str [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_register_machine: No attribute 'cloud' on Dict[str, Optional[str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_unregister_machine: No attribute 'assertTrue' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_unregister_machine_unavailable: No attribute 'port' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_set_machine_maintenace_state_false: No attribute 'node' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_node_set_provision_state: No attribute 'assertEqual' on Type[active_node_state] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_node_set_provision_state_wait_provide: No attribute 'cloud' on Type[managable_node_state] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_file_mock_is_readable_as_a_file: No attribute 'mock' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in get_themes_wp: No attribute 'encode' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_delete_router_not_found: Name 'r' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_list_router_interfaces_no_gw: Name 'ret' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_create_subnet_without_gateway_ip: Can't assign attribute 'return_value' on Dict[str, str] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_update_subnet_gateway_ip: No attribute 'cloud' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in GetAll: Name 'plugin_obj_list' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in PluginOutputExists: Name 'count' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_get_role: Name 'role' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_search_roles: Name 'roles' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_delete_role: Name 'role' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_list_role_assignments_v2: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_grant_revoke_role_user_domain: No attribute 'cloud' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_column: Name 'model_column' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in GetModule: Function imp.load_module was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in ProcessPlugins: No attribute 'PluginGroup' on Dict[str, bool] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in show_plugin_types: Name 'Plugin' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in parseMessage: unsupported operand type(s) for item retrieval: Dict[str, Any] and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in decorator: Function dict.update was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 13, in decorator: Function dict.update was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_create_server_with_create_exception: No attribute 'client' on Dict[str, Exception] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_create_server_with_get_exception: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in clear_data_oracle_map: No attribute 'parameter' on KeyError [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in remove_oracle_listener: No attribute 'Core' on ValueError [attribute-error]
File ""dummpy_input_file"", line 5, in remove_oracle_listener: No attribute 'parameter' on ValueError [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in output_instance: Name 'returner' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in version_tuple: unsupported operand type(s) for >>: 'print: Callable[[], None]' and 'TextIO' [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in Execute: Name 'mp' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _SplitEmails: No attribute 'strip' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in parse_args: Name 'state' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _process_args_for_args: Name 'arg' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in _match_long_opt: Name 'value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 15, in _match_short_opt: No attribute 'ignore_unknown_options' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in post: Name 'e' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in patch: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in delete: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in get: No attribute 'write' on Dict[str, bool] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in get: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in post: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in get: No attribute 'write' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in delete: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 18, in get: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in get: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in patch: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in get: No attribute 'get_component' on Dict[str, bool] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in get: Name 'filter_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in TransactionTableForURL: Name 'transaction' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in cmd: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in run: Name 'Args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in run: Name 'Content' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in run: Name 'Content' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_GetTypesForGroup_aux_should_return_a_list_with_its_types: Name 'aux_types' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_GetTypesForGroup_net_should_return_a_list_with_its_types: Name 'net_types' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_GetTypesForGroup_web_should_return_a_list_with_its_types: Name 'web_types' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_GetTypesForGroup_with_non_existing_group_should_return_empty_list_and_warning: Name 'types' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_Add_should_save_a_command_through_a_core_call: Name 'command_register' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in ProxyCheck: No attribute 'log_transactions' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in initialise_framework: unsupported operand type(s) for item retrieval: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in GetAll: No attribute 'target' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in PluginAlreadyRun: Name 'plugin_output_count' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: Name 'intrptr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in package_flask: Function os.listdir was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 10, in fetch_extensions_list: Name 'ext' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 19, in checkout_extension: Function posixpath.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in dump: No attribute 'file_path' on bytes [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 14, in load: Name 'dummy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in TopTransactionsBySpeed: Name 'transactions' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_meta_path_loader_without_is_package: Name 'app' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __getattr__: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in newext_simple: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in __init__: No attribute 'encode' on bytes [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in open_door: Name 'open_door_packet' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in available_configuration_files: Function genericpath.isfile was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in max_retries: Name 'n' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get: Name 'pathname' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in tearDown: Name 'test_path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in cluster_create_populate_test: No attribute 'validate_output' on List[str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in cluster_create_no_switch_test: No attribute 'assertEqual' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in cluster_create_vnodes_test: unsupported operand type(s) for item retrieval: 'f: TextIO' and ''num_tokens': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in validate: Name 'new_obj' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 15, in clone: No attribute 'cost' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 5, in __call__: No attribute 'params' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in save: No attribute 'state' on Callable[[int, Optional[types.FrameType]], Any] [attribute-error]
File ""dummpy_input_file"", line 8, in save: No attribute 'state' on int [attribute-error]
File ""dummpy_input_file"", line 8, in save: No attribute 'state' on signal.Handlers [attribute-error]
File ""dummpy_input_file"", line 8, in save: No attribute 'state' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in binVec2Int: unsupported operand type(s) for **: int and Callable[[Any, Any], Any] [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in format_pos: No attribute 'length' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in format_progress_line: No attribute 'width' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 37, in render_progress: No attribute 'color' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in _pipepager: No attribute 'stdin' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_editor: Name 'rv' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in edit_file: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in recv_comm: Name 'en_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 47, in Pull_Binary: Function int.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in make_logo: Name 'line' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in rebuild: Can't assign attribute 'list_of_hosts' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in update_lr: Name 'time' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in add_inputs: No attribute 'inputs' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 17, in merge_params: No attribute 'grads' on list [attribute-error]
File ""dummpy_input_file"", line 17, in merge_params: Can't assign attribute 'grads' on list [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __read_errors: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in __read: No attribute 'get' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in _load_current_cluster: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in __update_yaml: No attribute 'cluster' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _copy_agent: Name 'agent_source' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _stop_agent: No attribute 'close' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _write_agent_log4j_properties: No attribute 'write' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_file_can_be_closed: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in testSave: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _lookup_child: Name 'controller' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in option: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in ul: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in li: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in lead: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in textarea: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in script: No attribute 'pushResult' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in tr: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in td: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in next: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_client: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in handle: No attribute 'migrate_email_address' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in getMobilePages: No attribute 'getPages' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in getArticle: No attribute 'getCachedArticle' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in getLogo: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in generate_unique_username: Name 'ret' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in valid_email_or_none: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in import_attribute: Name 'attr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in import_callable: Name 'ret' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_current_site: Name 'site' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in clean: Name 'cleaned_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in doc_p: No attribute 'tag' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in css3_borderradius: Name 'css' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in css3_borderradiusbottomright: Name 'css' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in getMessage: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in getArticle: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in getChapter: No attribute 'getArticle' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in getFooter: No attribute 'blurb' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in getLogo: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in getPageTitle: No attribute 'blurb' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in getDescription: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in getKeyWords: No attribute 'newArticle' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in save: Name 'content' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in svg: No attribute 'pushResult' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _image: Name 'pixmap' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in div: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in p: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in h1: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in h2: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in h4: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in h5: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in h6: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in span: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in b: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in em: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in pre: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 2, in section: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in figure: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in figcaption: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in select: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in post: No attribute 'is_valid' on Dict[str, Any] [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_form_kwargs: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_success_url: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in closed: Name 'response_kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in post: No attribute 'get_context_data' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in AUTHENTICATION_METHOD: Name 'ret' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_form_kwargs: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _action_send: No attribute 'user' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_form_kwargs: Name 'kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get: No attribute 'get_redirect_url' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in post: No attribute 'get_redirect_url' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_context_data: No attribute 'redirect_field_name' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in check: No attribute 'create_command' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'admin_port' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 4, in create_command: No attribute 'executable' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in split_host: Name 'port' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in writetag: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in component_mobilenavigation: Name 'link' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in run: Name 'app' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in img: No attribute '_tag' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in hr: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in getExpire: 'time' object is not callable [not-callable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in getLoggedUserIds: unsupported operand type(s) for item retrieval: List[nothing] and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in asString: Name 't' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in linkBlock: No attribute 'BUILD_CSS' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in br: No attribute 'tag' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in run: Name 'err' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _action_TXNON: Name 'message' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in do_update_cert: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in run: Name 'url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in listTar: unsupported operand type(s) for +: List[str] and str [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in genFilesRaw: Name 'raw' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_can_join_declined: Name 'team' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_cannot_apply: Name 'team' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _uhex: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in saturate: No attribute 's' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in mix: Name 'color' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in delete_activity_on_subject_delete: No attribute 'activity_stream_item' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in created: Name 'dt' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in handle_reaction: Name 'score' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _hls2rgb: Name 'h' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in component_chapters: Name 'chapters' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in baseComponents: Name 'homePage' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in handleGetPostParams: No attribute 'form' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in getCookies: No attribute 'request' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
